prov/efa/src/rxr/rxr_pkt_type_rts.c

```diff
@@ -287,6 +287,36 @@ void rxr_pkt_handle_rts_sent(struct rxr_ep *ep, struct rxr_pkt_entry *pkt_entry)
 		rxr_inline_mr_reg(rxr_ep_domain(ep), tx_entry);
 }
 
+void rxr_pkt_handle_rts_send_completion(struct rxr_ep *ep, struct rxr_pkt_entry *pkt_entry)
+{
+	struct rxr_rts_hdr *rts_hdr;
+	struct rxr_tx_entry *tx_entry;
+
+	/*
+	 * for FI_READ, it is possible (though does not happen very offen) that at the point
```

Comment:
Might as well fix this typo while you're here.

Reply:
Thank you for pointing out! Fixed in new revision.